.github/workflows/CI.yaml

```
@@ -77,7 +78,7 @@ jobs:
         python -m pip install --upgrade pip setuptools
         pip install --upgrade mpi4py
         cd ../python_packaging/gmxapi
-        pip install .
+        pip install . --verbose
```

Oops. Not sure that first edit will work. Officially, options come before install target.
```suggestion
        pip install --verbose .
```